website/source/docs/agent/options.html.md

```diff
@@ -782,6 +782,9 @@ default will automatically work with some tooling.
       the maximum number of log entries that a server can trail the leader by before being considered unhealthy. Defaults
       to 250.
 
+    * <a name="min_quorum"></a><a href="#min_quorum">`min_quorum`</a> - Sets the minimum number of servers allowed in a cluster
+      before autopilot can prune dead servers. Defaults to bootstrap expect if set.
```

Comment:
Should this be `Defaults to bootstrap expect if not set`? Also, where in the code  does it default to bootstrap expect? 

Maybe it would be useful to add a test to verify the behavior when bootstrap expect and minquorum aren't equal.